proto/buf/alpha/registry/v1alpha1/resolve.proto

```diff
@@ -107,3 +113,30 @@ message GetLocalModulePinsResponse {
   // This includes the transitive deps.
   repeated buf.alpha.module.v1alpha1.ModulePin dependencies = 2;
 }
+
+message ResolveReferenceRequest {
+  // The plugin reference to resolve.
+  ResolveReferencePlugin plugin_reference = 1;
+  // The module reference to resolve.
+  LocalModuleReference module_reference = 2;
+}
+
+// ResolveReferencePlugin is a plugin reference.
+// If the version is empty, this is a reference to the latest version.
+message ResolveReferencePlugin {
```

Comment:
The name `PluginReference` conflicts with the type in https://github.com/bufbuild/buf/blob/4b05ef10993f94c52514c259cee664502bbbbc40/proto/buf/alpha/registry/v1alpha1/generate.proto#L22-L23 which is marked as deprecated.
As per alpha contract I assume this type can change names after release, but still support wire compatibility. 

This name will change name from `ResolveReferencePlugin` to `PluginReference` once the other file is deleted.